plugins/scaffolder/src/components/fields/OwnedEntityPicker/OwnedEntityPicker.tsx

```diff
@@ -38,98 +33,57 @@ export { OwnedEntityPickerSchema } from './schema';
  */
 export const OwnedEntityPicker = (props: OwnedEntityPickerProps) => {
```

Comment:
Should we perhaps mention this refactor in a changeset as well? Feeling it's good to have there in case this causes any issues.

Btw, this is a pure refactor right, no change in behavior?